packages/@webex/plugin-meetings/src/multistream/mediaRequestManager.ts

```diff
@@ -56,6 +57,8 @@ const CODEC_DEFAULTS = {
   },
 };
 
+const DEBOUNCED_SOURCE_UPDATE_TIME = 10;
```

Comment:
we probably need a bit bigger value than 10ms. The app will call size hint API a lot when the user resizes the app window, so maybe we should have 1000ms here?

Reply:
Changed to be 1000ms